src/components/SEP/Members/SEPMembers.tsx

```diff
@@ -234,13 +234,15 @@ const SEPMembers: React.FC<SEPMembersProps> = ({ sepId }) => {
         title={'SEP Secretary'}
         invitationUserRole={UserRole.SEP_SECRETARY}
       />
+      {/**
+       * Note: Do we actually need Formik here? It looks like it could be removed
+       *       and we can use regular React hooks and Mui form components
+       */}
```

Comment:
Maybe you are right there is no need for including formik here because we are not submitting anything. We are updating things one by one. I will check this.

Reply:
I see, then we can resolve it in a different ticket/PR.